platform/cavium/rules.mk

```diff
@@ -1,21 +1,13 @@
-CAVM_LIBSAI = libsai.deb
-$(CAVM_LIBSAI)_PATH = $(PLATFORM_PATH)/cavm_sdk
-CAVM_SAI = sai.deb
-$(CAVM_SAI)_PATH = $(PLATFORM_PATH)/cavm_sdk
-XP_TOOLS = xp-tools.deb
-$(XP_TOOLS)_PATH = $(PLATFORM_PATH)/cavm_sdk
-XPSHELL = xpshell.deb
-$(XPSHELL)_PATH = $(PLATFORM_PATH)/cavm_sdk
+include $(PLATFORM_PATH)/cavm-sai.mk
+include $(PLATFORM_PATH)/docker-syncd-cavm.mk
+include $(PLATFORM_PATH)/docker-orchagent-cavm.mk
 
-SONIC_COPY_DEBS += $(CAVM_LIBSAI) $(CAVM_SAI) $(XP_TOOLS) $(XPSHELL)
+SONIC_ALL += $(DOCKER_SYNCD_CAVM) \
+	     $(DOCKER_ORCHAGENT_CAVM)
```

Comment:
You'll also need [this line](https://github.com/Azure/sonic-buildimage/blob/master/platform/broadcom/rules.mk#L1) for building common dockers